tests/unit/test_session.py

```diff
@@ -0,0 +1,23 @@
+"""Session module tests."""
+
+import pytest
+
+from planet import DataClient, OrdersClient, SubscriptionsClient, Session
+from planet.exceptions import ClientError
+
+
+@pytest.mark.parametrize("client_name,client_class",
+                         [('data', DataClient), ('orders', OrdersClient),
+                          ('subscriptions', SubscriptionsClient)])
+def test_session_get_client(client_name, client_class):
+    """Get a client from a session."""
+    session = Session()
```

Comment:
According to our docs, [sessions](https://planet-sdk-for-python-v2.readthedocs.io/en/latest/python/sdk-guide/#create-a-session) should be used as a context manager or, if not, they should be closed explicitly with `session.close()`. For background on why, see the `Client` usage section of the httpx [docs](https://www.python-httpx.org/advanced/).

Reply:
while I'm not sure this matters for a test in terms of performance, etc, we want to be consistent with how we demonstrate usage, and tests are one of the places people could go to learn more about usage (I know it's something I do anyway).

Reply:
Yes!